app/views/editLiability/editLiability.scala.html

```diff
@@ -57,19 +57,14 @@
                         "CR" -> messages("ated.edit-liability.dispose-property.no")
                     ),
                     '_hintText -> messages("ated.edit-liability.p1"),
-                    '_legend -> messages("ated.edit-liability.header"),
+                    '_legend -> "",
```

Comment:
This line can be completely removed rather than set empty.

Reply:
Same as the other radio button.

Reply:
Same comment as above